index.bs

```diff
@@ -532,6 +532,9 @@ that agent.  Agents should discard any authentication message whose
 `auth-initation-token` is set and does not match the `at` provided by the
 advertising agent.
 
+If an agent chooses to show a user both a QR-code and a numeric PSK, they must
+be for the same PSK.
```

Comment:
Are agents limited to QR Code and numeric PSK? Couldn't it be encoded as a string or hex, or shown as a barcode? Why does it matter if it is the same PSK?

Reply:
I added a "why" and also generalized it to more than just 2 ways.  

I also changed it to a "should" since I suppose some auth mechanisms could allow for the displaying agent to accept either if they are different.